ubidots/ubidots_out.html

```diff
@@ -1,102 +1,149 @@
 <script type="text/x-red" data-template-name="ubidots_out">
-    <div class="form-row">
-        <label for="node-input-tier">
-            <i class="icon-flag"></i>
-            <span data-i18n="ubidots.select_tier"></span>
-        </label>
-        <select id="node-input-tier">
-            <option value="business">Ubidots</option>
-            <option value="educational">Ubidots for Education</option>
-        </select>
-    </div>
+   <div class="form-row">
+     <label for="node-input-tier" style="font-size:13px">
+       <i class="fa fa-flag"></i>
 
-    <div class="form-row">
-        <label for="node-input-name">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.name"></span>
-        </label>
-        <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
-    </div>
+       <span data-i18n="ubidots.select_tier"></span>
+      </label>
+       <select id="node-input-tier">
+         <option value="business">Ubidots</option>
+         <option value="educational">Ubidots for Education</option>
+     </select>
+   </div>
 
-    <div class="form-row">
-        <label for="node-input-token">
-            <i class="icon-lock"></i>
-            <span data-i18n="ubidots.token.label"></span>
-        </label>
-        <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
-    </div>
+   <div class="form-row">
+       <label for="node-input-name">
+         <i class="fa fa-tag"></i>
+         <span data-i18n="ubidots.name"></span>
+     </label>
+       <input type="text" id="node-input-name" data-i18n="[placeholder]ubidots.name" />
+   </div>
 
-    <div class="form-row">
-        <label for="node-input-device_label">
-            <i class="icon-tag"></i>
-            <span data-i18n="ubidots.device_label.label"></span>
-        </label>
-        <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
-    </div>
+   <div class="form-row">
+       <label for="node-input-token">
+         <i class="fa fa-lock"></i>
+         <span data-i18n="ubidots.token.label"></span>
+     </label>
+       <input type="text" id="node-input-token" data-i18n="[placeholder]ubidots.token.placeholder" />
+   </div>
+
+   <div class="form-row">
+       <label for="node-input-device_label">
+         <i class="fa fa-tag"></i>
+         <span data-i18n="ubidots.device_label.label"></span>
+     </label>
+       <input type="text" id="node-input-device_label" data-i18n="[placeholder]ubidots.device_label.placeholder" />
+   </div>
+  <div class="form-row">
+    <input type="checkbox" checked id="node-input-tls_checkbox" style="display: inline-block; width: auto; vertical-align: top;"></label>
```

Comment:
Same as in the other file, create classes for this and use them instead of using the inline styles of the node